### PR TITLE
Added javascript code for hiding previous dates

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Issue:[#316](https://github.com/ColoredCow/portal/issues/316)

Tasks:

- [x] To remove > from start date in edit file
- [x] To hide previous dates

### CODE REVIEW

**Comment by srivastava-jyoti** [comment]
@GauravGusain98 I have made a new PR for this issue. As in the previous one somehow other files got merged and I tried reverting but I was unable to revert.

For the issue which You pointed out that the changes are not visible on your local so for this we need to run `npm run watch`
in command prompt. I was facing same issue got checked on stackoverflow. Here's the link:[link](https://stackoverflow.com/questions/53425186/laravel-reactjs-changes-are-not-reflecting-after-change-in-my-jsreactjs-file)

Here's the screenshot from my local
Screenshot (57)

File: resources/js/app.js [hunk 1]
```diff
@@ -1001,6 +1001,20 @@ function datePickerChart(){
...
+	tdate = "0" + tdate;
+}
+var month = date.getMonth() + 1;
+if (month < 10) {
+	month = "0" + month;
+}
+var year = date.getUTCFullYear();
+var minDate = year + "-" + month + "-" + tdate;
+document.getElementById("start_date").setAttribute("min", minDate);
+document.getElementById("end_date").setAttribute("min", minDate);
```
Comment (GauravGusain98) [inline resources/js/app.js:1017]: Can you add a check here for the page where you need to load this script. You just need to add an id on the page and check the length of the id like this:

```suggestion
if ($('#page_id').length) {
    var date = new Date();
    var tdate = date.getDate();
    if (tdate < 10) {
                tdate = "0" + date;
    }
    var month = date.getMonth() + 1;
    if (month < 10) {
        	month = "0" + month;
    }
    var year = date.getUTCFullYear();
    var minDate = year + "-" + month + "-" + date;
    document.getElementById("start_date").setAttribute("min", minDate);
    document.getElementById("end_date").setAttribute("min", minDate);
}
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please do the suggested changes

File: resources/js/app.js (hunk 1 above)
Comment (srivastava-jyoti) [reply resources/js/app.js]: @GauravGusain98 I have made the required changes. And for the id part, I have used a unique id for start and end dates.These are not generic for the entire hr module. The previous dates are not visible for job description pages only.

**Review by GauravGusain98 — APPROVED** [review]
Looks good to me

**Pull request merged**